WasatchVCPPLib/WasatchVCPPLib/Uint40.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <mutex>
+
+namespace WasatchVCPP
+{
+    class Uint40 {
+    public:
+        uint16_t LSW;
+        uint16_t MidW;
+        uint8_t MSB;
+        uint8_t buf[3];
```

Comment:
I'm not sure if it's worth having the buf[] here.  If you were going to have it, I'd lean toward either having it of 5 bytes (where you could store the full 40-bit value, though that's not really needed), or 8 bytes (as apparently is needed when writing these payloads to firmware), but 3 bytes is a little confusing.  Anyway, it's not a bug, I'm just not sure it's useful and it might be worth removing buf[] altogether until we have a clear need for it (which will inform its size).